gobblin-compaction/src/main/java/gobblin/compaction/CompactionRunner.java

```diff
@@ -51,8 +51,8 @@
   private static final String DATAFORMAT_EXTENSION_NAME = ".dataformat.extension.name";
   private static final String OUTPUT = "output";
 
-  public static Properties properties = new Properties();
-  public static Properties jobProperties = new Properties();
+  static Properties properties = new Properties();
```

Comment:
They are both used in multiple other classes, e.g., HiveJdbcConnector, HdfsIO, SerialCompactor